src/classes/CW_BriefApprovalProcessTest.cls

```diff
@@ -0,0 +1,297 @@
+	    	nonApproverUser.UserRoleId = marketingRole.Id;
+	    	nonApproverUser.Department = 'Sport';
+	    	insert nonApproverUser;
+	    	
+	    	approverSetting = new ApproverListSetting__c(Name='1',Username__c=approverUser.Username,Role__c ='Marketing Ops' ,Type__c='Brief');
+	    	insert new List<ApproverListSetting__c>{approverSetting};
+		}
+	}
+	
+	public static void initData()
+	{
+        CW_Brand__c brand = TestDataUtil.createBrand();
+        insert brand;
+        
+        CW_Label__c label = TestDataUtil.createLabel(brand.Id);
+        insert label;
+        
+        CW_Country__c country = TestDataUtil.createCountry();
+        insert country;
+        
+        CW_Product__c product = TestDataUtil.createProduct();
+        insert product;
+        
+        CW_Language__c language = TestDataUtil.createLanguage();
+        insert language;
+        
+        CW_Site_Catalogue__c siteCatalogue = TestDataUtil.createSiteCatalogue(label.Id, country.Id, product.Id);
+        insert siteCatalogue;
+    	
+     	camp = TestDataUtil.createCampaign();
+     	insert camp;
+     	 	
+	    CW_Campaign_Sites__c campSites = TestDataUtil.createCampaignSite(camp.Id, siteCatalogue.Id, language.Id);
+       	insert campSites;
+	    	
+        
+	}
+	
+	public static void updateCampRequiredFields(CW_Campaign__c campaign)
+	{
+        campaign.Suggested_MarCom_Delivery_Date__c = Date.today().addDays(10);
+    	campaign.Main_Campaign_Business_Objective__c = 'test';
+    	campaign.Campaign_Focus__c = 'Retention'; 
+    	campaign.What_is_being_communicated__c ='test';
+    	campaign.Target_Audience_Insight__c = 'test';
+    	campaign.Urgent_Campaign__c = false;
+    	update campaign;
+	    
+	}
+
+    static testMethod void submitBriefForApprovalNoCampaignSite() 
+    {
+    	initUsersAndCustomSettings();
+    	
+    	CW_Campaign__c newCamp = TestDataUtil.createCampaign();
+    	newCamp.OwnerId = nonApproverUser.Id;
+	    insert newCamp;
+	    
+	    updateCampRequiredFields(newCamp);
+    		
+     	System.runAs(nonApproverUser)
+     	{
+     		updateCampRequiredFields(newCamp);	
+	    	
+	        newCamp = [SELECT Name, Campaign_Start_Date__c,
+	        			   Campaign_End_Date__c, 
+	    				   Suggested_MarCom_Delivery_Date__c,
+	    				   OwnerId, 
+	    				   Main_Campaign_Business_Objective__c,
+	    				   Campaign_Focus__c,Roadmap__c,
+	    				   What_is_being_communicated__c,
+	    				   Target_Audience_Insight__c,
+	    				   Requestor_Department__c,
+	    				   Cost_Approval_Status__c,
+	    				   Urgent_Campaign__c
+	    			FROM CW_Campaign__c
+	    			WHERE Id =: newCamp.Id];
+	        
+	        CW_BriefApprovalProcess briefAP = new CW_BriefApprovalProcess();
+	        briefAP.checkBriefForApproval(newCamp);
+
+	        system.assertEquals(briefAP.submitBriefForApproval , false);
+        	
+     	 }
+    }
+    
+    static testMethod void submitBriefForApprovalNoRoadmapApproved() 
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility